src/Lexer.js

```diff
@@ -328,12 +328,13 @@ module.exports = class Lexer {
   /**
    * Lexing/Compiling
    */
-  inlineTokens(src, tokens = [], inLink = false, inRawBlock = false, prevChar = '') {
+  inlineTokens(src, tokens = [], inLink = false, inRawBlock = false) {
```

Comment:
@calculuschild I don't think we actually ever send prevChar to this function. Is there a reason this was added to the function signiture in #1686?

I changed it to get prevChar from the tokens below.

Reply:
No, it was likely an accidental remnant from an earlier commit I didn't fully clean up. It doesn't need to be there.